### DISCUSSION TITLE
🇮🇳 List of resources in Hindi language

### DISCUSSION THREAD

**Discussion by SkalskiP** [SkalskiP/courses#18] [Ideas]
English is not the only language in the world. Many people do not speak English. Many creators do not make content in English. Therefore, we want to create separate lists of resources that can help people who want to build ML knowledge and do not speak that language. **Hindi** is the first language we want to support. 

I am looking for people who would like to help me. I don't know Hindi 😞 I need your help to make it happen.

**Comment by Jaysheel11** [comment]
India has a huge ML community. And an organized list of materials will help the beginners and others to easily locate the resources that they need to reach at Intermediate level or even Advance level . I am looking forward to contribute here.

**Reply by SkalskiP** [reply]
Awesome, @Jaysheel11 🔥! Can we start to assemble the list of resources in this thread?

**Reply by SkalskiP** [reply]
@Jaysheel11 would you be kind enough and create PR where you translate this [README.md](https://github.com/SkalskiP/courses/blob/master/README.hindi.md) to Hindi?

**Reply by Jaysheel11** [reply]
Yes

**Reply by SkalskiP** [reply]
Ideally, everything in that file should be in Hindi. The only word that should be in English is the word "English" in the header and maybe the title. The rest should be Hindi 💪🏻 ! This way people will know that Hindi support is coming.

**Reply by Jaysheel11** [reply]
In India actually title of vids and heading and description all are in English but the language spoken in vid is hindi. So its better we keep everything in english and only the language spoken in vid should be in hindi.

**Reply by SkalskiP** [reply]
but the text inside `👋 hello`, `🚧 help us` and `🦸 contribution` sections should be in Hindi?

**Reply by Jaysheel11** [reply]
Nup only the language spoken inside the vid should be in Hindi .As a Indian student who has watched countless tutorials I have seen its better that the name, title and description should be in English because it helps to easily search the particular topic easily .In Indian universities also the books or material they share with us are in English only. But you know its easy to understand in  when someone communicate with you in your mother tongue or the language you use in your daily life.

**Reply by SkalskiP** [reply]
Taking all that into consideration would you make any changes to that `README.hindi.md` file?

**Reply by Jaysheel11** [reply]
can we change creme de la creme I am pretty sure most of people in India will not be able to figure out the meaning of creme de la creme

**Reply by SkalskiP** [reply]
We can drop it in the Hindi version. No problem.

**Reply by Jaysheel11** [reply]
sarvashreshth mein se sarvottam - its hindi written in english 
सर्वश्रेष्ठ में से सर्वोत्तम - this one is written in hindi

**Reply by Jaysheel11** [reply]
https://youtube.com/playlist?list=PLKnIA16_Rmvbr7zKYQuBfsVkjoLcJgxHH hey this playlist is very good.It explains how to  Machine learning algo works and other different techniques and also how whole production works.Please check it out title is in english

**Reply by SkalskiP** [reply]
I took a look at a few videos. Looks cool. It looks like he is using a mix of English and Hindi. Am I right?

**Reply by Jaysheel11** [reply]
its purely hindi as i told you we have used few words in english like you say hola sometimes to greet someone

**Reply by Jaysheel11** [reply]
sorry for late response met with an accident due to heavy winds in my area

**Comment by Masterhazi** [comment]
are we still working on translating to hindi?

**Comment by Jaysheel11** [comment]
Hey I stopped working on it?.Is this project started again?

________________________________
